lib/formatters/csv.js

```diff
@@ -38,7 +38,38 @@ function formatCsv (sites, options = {}) {
     });
   });
 
-  return toCsvString(rows);
+  const sortedRows = rows.sort((a, b) => {
```

Comment:
The implementation for this needs to not live entirely inside `formatCsv()`.

1. It’s a fairly abstract operation so you should separate it, because sooner or later you’ll run into the…
2. Practical realities! `query-db-and-email` [does not use `formatCsv()` directly](https://github.com/edgi-govdata-archiving/web-monitoring-versionista-scraper/blob/master/bin/query-db-and-email#L169-L196) because the data it is formatting is a completely different structure from what is pulled directly out of Versionista. If the sorting method is separate, it can be called from there.